sklearn/model_selection/_search.py

```diff
@@ -782,6 +785,20 @@ def _store(key_name, array, weights=None, splits=False, rank=False):
                     results["split%d_%s"
                             % (split_i, key_name)] = array[:, split_i]
 
+            # Compute ttest between each candidates pair
+            if ttest:
+                matrix_pval = MaskedArray(np.empty([n_candidates,
+                                                    n_candidates]),
+                                          mask=True)
+                for cand_i in range(n_candidates):
+                    for cand_k in range(cand_i):
```

Comment:
use `for cand_i, cand_k in itertools.combinations(n_candidates, 2):`

Reply:
> The documentation of cv_results_ needs updating to make the format of the output clearer.

Yes indeed. I was waiting for feedback on whether that is the right output or not, but I can commit a description of the current option for now.